app/scripts/modules/titus/pipeline/stages/bake/titusBakeStage.js

```diff
@@ -77,7 +77,7 @@ module.exports = angular.module('spinnaker.core.pipeline.stage.titus.titusBakeSt
 
       let buildParams = $.param(stage.repository.buildParameters);
       if (buildParams.length > 0) {
-        url += '&' + buildParams;
+        url += '&' + decodeURIComponent(buildParams);
```

Comment:
You have a specific usage of things that need to be encoded in the buildParams?
The entire repo url itself isn't encoded but only buildParams.